README.md

```diff
@@ -285,6 +285,10 @@ The `setting` option can take any of the following values:
 Keep in mind that this configuration must mirror the
 [configuration of openwisp-radius (OPENWISP_RADIUS_OPTIONAL_REGISTRATION_FIELDS)](https://openwisp-radius.readthedocs.io/en/latest/user/settings.html#openwisp-radius-optional-registration-fields).
 
+#### Configure Social Login
+
+In order to authentication to work social login has to be configured. Follow this for more info: https://openwisp-radius.readthedocs.io/en/latest/user/social_login.html
+
```

Comment:
```suggestion
#### Configure Social Login

You will need to configure social login to enable users login using third-party services. Refer to ["Social Login" section of OpenWISP Radius documentation](https://openwisp-radius.readthedocs.io/en/latest/user/social_login.html) for detailed setup instructions. 
```

I am not sure but this sounds better. Let someone else run it through before making this change. 😄

Reply:
Yes, this sounds much better, and the hyperlink in README makes representation good.